constants.js

```diff
@@ -0,0 +1,12 @@
+
+module.exports  = {
+    URL_LIST :{
+        BASE_URL : 'https://api.instagram.com/v1/users',
+        AUTH_URL : 'https://api.instagram.com/oauth/authorize/?client_id=609f4f8ae1e44d3aa091ac0f05ae1477&redirect_uri=http://127.0.0.1:8081&response_type=token',
+        OWNER_INFO_URL : '/self/?access_token=',
+        OWNER_RECENT_MEDIA_URL : '/self/media/recent/?access_token='
+    },
+    ACCESS_TOKEN  : '1642314104.609f4f8.28c2ae0f396e4a0dbf8c64ff5d6a9037'
```

Comment:
We'll be storing base URL in `Controller.js` and access-token in `Login.js` and API endpoint URL's will be typed during XHR requests so I don't see a need of having this constants file.

Reply:
To make API calls in a smart way we should keep all the endpoints in globally shared file. Most costliest operation in web development is DOM manipulation. By keeping URL in different component and passing it from parent to child and child to parents is cause two issue
1) small performance issue
2) readability and re-usability issue. - Suppose instead of X URL we need to change Y URL then we need to change all file. But in Case of constants only one file need to change.

Reply:
Makes sense.